src/UI/Activity/Queue/TimeleftCell.js

```diff
@@ -10,24 +10,39 @@ module.exports = NzbDroneCell.extend({
         this.$el.empty();
 
         if (this.cellValue) {
-            if (this.cellValue.get('status').toLowerCase() === 'pending') {
-                var ect = this.cellValue.get('estimatedCompletionTime');
-                var time = '{0} at {1}'.format(FormatHelpers.relativeDate(ect), moment(ect).format(UiSettingsModel.time(true, false)));
+            var status = this.cellValue.get('status').toLowerCase();
+            var ect = this.cellValue.get('estimatedCompletionTime');
+            var time = '{0} at {1}'.format(FormatHelpers.relativeDate(ect), moment(ect).format(UiSettingsModel.time(true, false)));
+
+            if (status === 'pending') {
                 this.$el.html('<div title="Delaying download till {0}">-</div>'.format(time));
-                return this;
-            }
+            } else if (status === 'downloadclientunavailable') {
+                this.$el.html('<div title="Retrying download at {0}">-</div>'.format(time));
```

Comment:
We're throwing every download on the pile, might confuse the user since it's not really gonna download all 20 releases.

Tbh adding Fallback immediately might be preferable, since then you can just mark the first as downloadclientunavailable and the rest as Fallback.